src/uct/tcp/tcp.h

```diff
@@ -28,7 +28,7 @@ typedef unsigned (*uct_tcp_ep_progress_t)(uct_tcp_ep_t *ep);
  */
 typedef struct uct_tcp_am_hdr {
     uint8_t                       am_id;
-    uint16_t                      length;
+    unsigned                      length;
```

Comment:
Better to use types with fixed size for wired protocols, i.e. uint32_t

Reply:
Yes, I agreed that it is better. But `uct_ep_am_short` uses unsigned for length, I wanted to align them.
@yosefe wdyt?

Reply:
uint32_t

Reply:
@evgeny-leksikov @yosefe Thanks! Fixed